.github/workflows/multicluster.yaml

```diff
@@ -4,7 +4,7 @@ on:
   pull_request: {}
   push:
     branches:
-      - master
+      - pr/tgraf/fix-clustermesh
```

Comment:
I guess this change needs to be reverted before merging.

Reply:
Yes, instead of disabling GitHub actions, we'll need to change the branch trigger in the future so we can test re-enabling in a particular PR.